Pod/Classes/OAStackView.m

```diff
@@ -16,6 +16,7 @@
 
 @interface OAStackView ()
 @property(nonatomic, copy) NSArray *arrangedSubviews;
```

Comment:
No, this property is used in line 76. It's provided to be consist with iOS9 API.